### DISCUSSION TITLE
Dart/Flutter SDK

### DISCUSSION THREAD

**Discussion by apstanisic** [directus/directus#3404] [Feature Requests]
With Flutter gaining in popularity it would be cool if we had official Dart SDK. I created placeholder `directus` package, and can develop sdk. Just wanted to check with you if you're willing for source to be hosted in directus organization, and to transfer ownership of the package? 
[https://pub.dev/packages/directus](https://pub.dev/packages/directus)

**Comment by rijkvanzanten** [comment]
Sure thing! We won't have the resources to "officially" support that SDK (small core team), but I can set you up with a repo under the org if you wanna steer it 🙂

**Comment by apstanisic** [comment]
Sure. Do you want to create publisher on pub.dev (same as organization on npm), or should it remain published under my account?

**Reply by benhaynes** [reply]
Hey! I created an approved _publisher_ account on pub.dev ... and invited your account/email. Let me know if you have any issues connecting or moving the package into it. Thanks again!! 👍

**Reply by apstanisic** [reply]
Everything works. Just published test update. 🎉

**Reply by benhaynes** [reply]
Nice!! Confirmed on my side too. Do we want/need to include this anywhere so it can be used for installs? What's the best way to add this to the Docs? @rijkvanzanten

**Reply by NilsBaumgartner1994** [reply]
Any update how far the package became?

**Reply by paescuj** [reply]
> Any update how far the package became?

The package itself seems to be actively maintained:
https://pub.dev/packages/directus (repository at https://github.com/apstanisic/directus-dart)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
